adhocracy-plus/assets/scss/components/_project_tile_carousel.scss

```diff
@@ -68,10 +69,6 @@
 @media screen and (min-width: $breakpoint-md) {
     .project-tile-carousel {
         margin: $spacer 0;
-
-        .tile.organisation__tile {
-            height: 570px;
-        }
```

Comment:
this might be why they look weird on organisation page now?

Reply:
just checked, it is inside of `.project-tile-carousel` so no effect on organisation